src/assets/stylesheets/ui-root.scss

```diff
@@ -51,3 +51,18 @@ body.vr-mode {
   z-index: 11;
   pointer-events: auto;
 }
+
+:local(.new-loader-refresh-prompt) {
+  @extend %default-font;
+  background: theme.$background1-color;
+  border: 3px solid theme.$border1-color;
+  border-radius: 16px;
+  color: theme.$text1-color;
+  left: 50%;
+  max-width: 70%;
+  padding: 20px;
+  position: absolute;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 100;
+}
```

Comment:
I'm not a designer. Comments to improve these are very welcome.

Reply:
this looks good to me. we also have toast component in the lilypad component library and a modal if you are interested..

https://mozilla.github.io/lilypad/?path=/story/example-notification--main

https://mozilla.github.io/lilypad/?path=/story/example-modal--main

but again this is probably fine.

Reply:
Thanks I'm thinking of merging as is for now.